### PR TITLE
BaSP-TG-90: Modify NavBar to follow Figma design

### PR DESCRIPTION
**Pull request by SuarezAlexisEzequiel**
- Items should be display in column
- Follow the design from Figma
- We should have it on the left
- For now we don't need the user avatar
- Should be display on every page

### CODE REVIEW

File: src/Components/NavBar/NavBar.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import userIcon from '../../Assets/user.svg';
+import styles from './navBar.module.css';
+
+const NavBar = (props) => {
+  if (window.location.pathname == '/home' || window.location.pathname == '/nav') {
```
Comment (cocolobos) [inline src/Components/NavBar/NavBar.jsx:7]: we should use history.push from react router as shown in class

File: src/Components/NavBar/NavBar.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      <nav className={styles.navbar}>
+        <div className={styles.userInfo}>
+          <img src={userIcon} className={styles.userImg} />
+          <div className={styles.userName}>User Name</div>
+          <div className={styles.userRole}>User Role</div>
+        </div>
+        <div className={styles.menu}>Menu</div>
+        <ul className={styles.rutes}>
+          <li>
+            <button onClick={() => props.history.push('/projects/')}>projects</button>
```
Comment (cocolobos) [inline src/Components/NavBar/NavBar.jsx:20]: this should be Link component from react router

File: src/Components/NavBar/navBar.module.css [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  justify-content: flex-start;
+  align-items: flex-start;
+}
+
+.rutes > li {
+  margin: 0 0 1.5vh 0;
+  list-style: none;
+}
+
+.rutes > li > button {
```
Comment (cocolobos) [inline src/Components/NavBar/navBar.module.css:67]: can we add color on hover and cursor:pointer  ?

File: src/Components/NavBar/navBar.module.css [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  width: 100%;
+  padding: 0;
+  font-size: 0.75rem;
```
Comment (cocolobos) [inline src/Components/NavBar/navBar.module.css:70]: I think this could be bigger what do you think ?

File: src/Components/NavBar/NavBar.jsx (hunk 1 above)
Comment (SuarezAlexisEzequiel) [reply src/Components/NavBar/NavBar.jsx:7]: The conditional is coded like this to refer to the pathname and not to redirect

File: src/Components/NavBar/navBar.module.css (hunk 4 above)
Comment (SuarezAlexisEzequiel) [reply src/Components/NavBar/navBar.module.css:70]: Sure, not problem

File: src/Components/NavBar/navBar.module.css (hunk 3 above)
Comment (SuarezAlexisEzequiel) [reply src/Components/NavBar/navBar.module.css:67]: Yes, we can

File: src/Components/NavBar/NavBar.jsx (hunk 2 above)
Comment (SuarezAlexisEzequiel) [reply src/Components/NavBar/NavBar.jsx]: Done

File: src/Components/NavBar/NavBar.jsx (hunk 1 above)
Comment (SuarezAlexisEzequiel) [reply src/Components/NavBar/NavBar.jsx]: Changed to 'location.pathname'

**Review by cocolobos — APPROVED** [review]

**Pull request merged**